src/runtime/storage/firebase/firebase-storage.ts

```diff
@@ -552,7 +552,7 @@ class FirebaseVariable extends FirebaseStorageProvider implements VariableStorag
     return this.set(null, originatorId, barrier);
   }
 
-  async cloneFrom(handle) {
+  async cloneFrom(handle): Promise<void> {
```

Comment:
Not void?

Reply:
async functions return promises, thus the Promise<void> signature.

This typing catches cases where someone might return values incorrectly.